modules/rte_mis_school/config/install/field.storage.taxonomy_term.field_aid_status.yml

```diff
@@ -0,0 +1,26 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - taxonomy
+id: taxonomy_term.field_aid_status
+field_name: field_aid_status
+entity_type: taxonomy_term
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: aided
+      label: Aided
+    -
+      value: unaided
+      label: unaided
+  allowed_values_function: ''
```

Comment:
Instead of defining it here, Can we add these values in `rte_mis_school.settings.yml` and get the option values using a function call?

Reply:
Updated